core/controllers/base.py

```diff
@@ -59,16 +59,19 @@
 
 
 @functools.lru_cache(maxsize=128)
-def load_template(filename):
+def load_template(filename, template_is_aot_compiled):
     """Return the HTML file contents at filepath.
 
     Args:
         filename: str. Name of the requested HTML file.
+        template_is_aot_compiled: bool. Used to determine which bundle to use.
```

Comment:
AOT compilation will be the default behavior at the end of migration. Why not consider that a webpage is aot compiled by default and we use a flag to specify that it is not?